composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=7.0.0"
```

Comment:
Could you please use [class_alias](http://php.net/manual/en/function.class-alias.php) instead of dropping support of 5.x

Reply:
Done ;)